python/README.md

```diff
@@ -10,4 +10,8 @@
 
   * Python package is for Python > 3.4
 
+## Compile protobuf
 
+```
+python -m grpc_tools.protoc -I${PWD}/seldon_core/proto --python_out=${PWD}/seldon_core/proto --grpc_python_out=${PWD}/seldon_core/proto --mypy_out=${PWD}/seldon_core/proto ${PWD}/seldon_core/proto/prediction.proto
```

Comment:
Ideally, we should only reference the existing Makefile target so we can track it only in one place.